### ISSUE TITLE
Recall@K

### ISSUE THREAD

**Issue by deychak** [tensorflow/recommenders#533]
I'm using the TFRS Retrieval model for implicit feedback. I'm following the basic guide online. Is it possible to provide other metrics, e.g. Recall@K, on top of the default TopKCategoricalAccuracy? I tried to provide the Keras Recall@K metric to `model.compile` but I see nothing printed out for it from the Retrieval TFRS Two Tower model during`model.fit` and during `model.evaluate`.

**Comment by deychak** [comment]
I ended up writing my own recall@k implementation similar to what I later found towards the bottom half of this [tutorial](https://www.tensorflow.org/recommenders/examples/efficient_serving).
